pkg/webhook/resources/pod/mutator.go

```diff
@@ -17,16 +17,22 @@ import (
 	"github.com/harvester/harvester/pkg/webhook/types"
 )
 
-var matchingLabels = []labels.Set{
-	{
-		"longhorn.io/component": "backing-image-data-source",
+var matchingLabelsForNamespace = map[string][]labels.Set {
+	"longhorn-system": {
+		{
+			"longhorn.io/component": "backing-image-data-source",
+		},
 	},
-	{
-		"app.kubernetes.io/name":      "harvester",
-		"app.kubernetes.io/component": "apiserver",
+	"harvester-system": {
+		{
+			"app.kubernetes.io/name":      "harvester",
+			"app.kubernetes.io/component": "apiserver",
+		},
 	},
-	{
-		"app": "rancher",
+	"cattle-system": {
+		{
+			"app": "rancher",
+		},
```

Comment:
Can we use the `const` namespaces defined in the `util` package? E.g., https://github.com/harvester/harvester/blob/90bac098aef17a8eea1fd7fc40341a85ae86538e/pkg/util/constants.go#L88